src/serializer/visitors.js

```diff
@@ -72,7 +72,7 @@ export let ClosureRefReplacer = {
     state.requireStatistics.count++;
     if (state.modified[path.node.callee.name]) return;
 
-    let moduleId = path.node.arguments[0].value;
+    let moduleId = "" + path.node.arguments[0].value;
```

Comment:
Further investigations on this can happen later. I filed an issue for it: https://github.com/facebook/prepack/issues/852